user-manual/transfer/forensic.rst

```diff
@@ -4,130 +4,169 @@
-      can choose YES or NO.
+      Click on the report button to add metadata
 
-During this workflow, some deviations from standard microservice behaviors
-occur:
+#. Enter your metadata and click **Save**, then close the tab.
 
-1. The Characterize and extract metadata microservice runs a tool called fiwalk
-   on digital forensic images.
+   .. figure:: images/forensic-metadata-template.*
+      :align: center
+      :figwidth: 70%
+      :width: 100%
+      :alt: Forensic disk image metadata template
 
-2. The Examine contents microservice allows you to choose whether you would like
-   to run a tool called bulk extractor, which identifies and outputs text
-   reports about PII information contained in the set of objects. These reports
-   can be analyzed later using other tools such as BitCurator. In Archivematica,
-   the reports are contained in the logs folder of the transfer, and later the
-   logs folder for each SIP that all or part of the transfer is contained
-   within.
+      Fill in the metadata template and click save.
+
+   .. important::
+
+      The metadata form opens in a new tab. After clicking save, you **must** close
+      the new tab to go back to your in-progress transfer, rather than clicking on
+      the Transfer tab from the metadata form.
+
+#. If you are planning to start multiple transfers, you can enter metadata for
+   the next image by clicking on **Add Next**, located to the right of **Start
+   Transfer**.
+
+   .. figure:: images/forensic-add-next.*
+      :align: center
+      :figwidth: 70%
+      :width: 100%
+      :alt: Add the metadata for the next image by clicking Add Next
+
+      Add the metadata for the next image by clicking Add Next
+
+#. Once all images are loaded to the dashboard and all metadata is added, select
+   **Start Transfer**.
+
+#. Proceed through the normal :ref:`Transfer <transfer>` and
+   :ref:`Ingest <ingest>` workflows.
 
 .. note::
 
-   This microservice can be run effectively on ALL transfer types, they do not
-   have to be digital forensic disk images.
+   Note that during the *Characterize and extract metadata* microservice,
+   `fiwalk`_ will be used on forensic disk image files.
+
+.. _compound-disk-images:
 
-For simple image transfers, continue directly into the Ingest workflow and
-finish processing using the standard :ref:`Archivematica processing instructions
-<ingest>`.
+Compound disk images
+++++++++++++++++++++
+
+You can combine multiple parts of a compound disk image into a single AIP, if
+desired, by using the backlog arrangement functionality in Archivematica.
+
+#. Start each part of the compound disk image as a single transfer, as per the
+   instructions above.
+
+#. When you reach the *Create SIP* job, select "Send to backlog". Do this for
+   each transfer.
+
+#. In the :ref:`Appraisal tab <appraisal>`, use the
+   :ref:`Arrangement <arrangement>` pane to combine your transfers into one SIP.
```

Comment:
You tested this, right? :grimacing: :sweat_smile: hahaha...

Reply:
I did not but it... should... work? Gahhhhh.